geode-apis-compatible-with-redis/README.md

```diff
@@ -329,9 +329,10 @@ start server \
 |                    	|                                                    	| ZREVRANK                      	|
 |                    	|                                                    	| ZSCAN                         	|
 |                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
+|                    	|                                                    	| ZUNIONSTORE                   	|
 
 **NOTES:**
 
 [1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
 [2] - SLOWLOG is implemented as a NoOp.
+[3] - Redis accepts 64-bit signed integer for the cursor and COUNT parameters be represented as an integer for the HSCAN command. This operation is limited to 32-bit integer values.
```

Comment:
Was this meant to be something like:
```
Redis accepts 64-bit signed integers for the HSCAN cursor and COUNT parameters. The Geode APIs compatible with Redis only accept 32-bit integers for these parameters.
```

Reply:
wow I completely mangled that text. idk what happened